modules/navier_stokes/src/fvkernels/INSFVTurbulentDiffusion.C

```diff
@@ -18,18 +18,22 @@ INSFVTurbulentDiffusion::validParams()
   InputParameters params = FVDiffusion::validParams();
   params.addClassDescription(
       "Computes residual for the turbulent scaled diffusion operator for finite volume method.");
-  params.addParam<MooseFunctorName>("scaling_coef", 1.0, "diffusion coefficient");
+  params.addParam<MooseFunctorName>(
+      "scaling_coef", 1.0, "Scaling factor to divide the diffusion coefficient with");
   params.set<unsigned short>("ghost_layers") = 2;
   params.addParam<std::vector<BoundaryName>>(
       "walls", {}, "Boundaries that correspond to solid walls.");
+  params.addParam<bool>(
+      "newton_solve", false, "Whether a Newton method is used to solve the equations");
 
   return params;
 }
 
 INSFVTurbulentDiffusion::INSFVTurbulentDiffusion(const InputParameters & params)
   : FVDiffusion(params),
     _scaling_coef(getFunctor<ADReal>("scaling_coef")),
-    _wall_boundary_names(getParam<std::vector<BoundaryName>>("walls"))
+    _wall_boundary_names(getParam<std::vector<BoundaryName>>("walls")),
+    _newton_solve(getParam<bool>("newton_solve"))
```

Comment:
I think we could decide this automatically by casting the executioner.

Reply:
To either steady, transient, multisystem steady etc. Seems easier to just ask the user

Reply:
No, if it is not inheriting from segregated solver base.

Reply:
I created https://github.com/idaholab/moose/issues/28814
I ll fill it up more later.

I dont think a cast is a viable solution, it would not catch other executioners in downstream applications.

Reply:
I am just saying that right now anything else that is not inheriting from segregated solver base is newton based. Especially for the physics this kernel is supposed to add. If you don't think casting is the right way that is fine with me. Then please add another method to check if the user got this parameter right. I don't want them using the newton solver options with SIMPLE and vice versa.

Reply:
Seems like I ll solve #28814 here then